src/components/organisms/AdminVenueView/AdminVenueView.tsx

```diff
@@ -1,65 +1,60 @@
-import React, { useCallback, useMemo, useState } from "react";
-import { Button, Nav } from "react-bootstrap";
-import { Link } from "react-router-dom";
+import React, { useMemo, useState } from "react";
+import { Nav } from "react-bootstrap";
 import classNames from "classnames";
 
-import AdvancedSettings from "pages/Admin/AdvancedSettings";
-import BasicInfo from "pages/Admin/BasicInfo";
-import EntranceExperience from "pages/Admin/EntranceExperience";
-import VenueDetails from "pages/Admin/Venue/Details";
```

Comment:
You are correct, `BasicInfo` is no longer being used and `VenueWizard` is used instead. It can be removed the the other screen can be adapted/changed if needed.

`VenueDetails` is being used in one more place. `Admin.tsx`

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1511/commits/767b8397b022e9e276ed46c3f1bd384570fa4751